src/tools/perf/lib/libperf.c

```diff
@@ -1323,7 +1338,7 @@ static size_t ucp_perf_pack_daemon_info(const ucx_perf_context_t *perf,
 
 static size_t ucp_perf_daemon_info_size(const ucx_perf_context_t *perf)
 {
-    return sizeof(uint64_t) + sizeof(struct sockaddr_storage) +
+    return sizeof(uint64_t) + UCS_IPV6_SOCKADDR_LEN +
```

Comment:
why IPv6?

Reply:
This function returns buffer size large enough to hold the message to the daemon.
Since address that we pass can be either IPv4 or IPv6 or NULL, I decided to use the larger size that would fit all cases. Since the diff between the 2 is just 12 bytes, I thought it's not a big deal if we allocate a bit more just in case. This way we don't need to check whether address is IPv4/IPv6/empty in advance.
I think the same pattern is used in rkey serialization

If you believe it makes sense to calculate the precise buffer length before allocation - let me know, I'll modify the code accordingly

Reply:
let's see how complex it is to pass exact address size. maybe reuse that size value also in the send function

Reply:
ok